catalog-rest-service/src/main/resources/json/schema/operations/pipelines/airflowPipeline.json

```diff
@@ -46,9 +26,113 @@
             "databaseServiceMetadataPipeline",
             "databaseServiceQueryUsagePipeline"
           ]
+        },
+        "sourceConfig": {
+          "description": "A connection to a dashboard service, database service or messaging service.",
+          "$ref": "../../entity/services/serviceConnection.json#/definitions/serviceConnection"
+        },
+        "sinkConfig": {
+          // let's do the same approach as with the source, currently just with the metadata-rest
         }
       },
       "additionalProperties": false
+    },
+    "airflowConfig": {
+      "description": "Properties to configure the Airflow pipeline that will run the workflow.",
+      "type": "object",
+      "javaType": "org.openmetadata.catalog.operations.pipeline.AirflowConfig",
+      "properties": {
+        "forceDeploy": {
+          "description": "Deploy the pipeline by overwriting existing pipeline with the same name.",
+          "type": "boolean",
+          "default": "true"
```

Comment:
We need to fix these values to not be generated as string but as proper booleans